lib/services.cf

```diff
@@ -139,6 +139,9 @@ bundle agent standard_services(service,state)
 # methods:
 #     "SSHD should be running" usebundle => standard_services("sshd", "start");
 # ```
+#
+# **Notes:**
+# FreeBSD uses "one" prefixed commands, e.g. onstatus, onestart, onestop so that services that are not enabled can still be managed
```

Comment:
maybe?

```suggestion
# FreeBSD uses "one" prefixed commands, e.g. onestatus, onestart, onestop so that services that are not enabled can still be managed
```